react-native.config.js

```diff
@@ -1,10 +1,32 @@
-const fs = require('fs');
-const path = require('path');
-const {
-  androidManifestPath,
-  iosProjectPath,
-  windowsProjectPath,
-} = require('react-native-test-app');
+const project = (() => {
+  const fs = require('fs');
+  const path = require('path');
+  try {
+    const {
+      androidManifestPath,
+      iosProjectPath,
+      windowsProjectPath,
+    } = require('react-native-test-app');
+    return {
+      android: {
+        sourceDir: path.join('example', 'android'),
+        manifestPath: androidManifestPath(
+          path.join(__dirname, 'example', 'android')
+        ),
+      },
+      ios: {
+        project: iosProjectPath('example/ios'),
+      },
+      windows: fs.existsSync('example/windows/AsyncStorageExample.sln') && {
+        sourceDir: path.join('example', 'windows'),
+        solutionFile: 'AsyncStorageExample.sln',
+        project: windowsProjectPath(path.join(__dirname, 'example', 'windows')),
+      },
+    };
+  } catch (_) {
+    return undefined;
```

Comment:
so what's exactly issue here? why config might throw?

Reply:
If you're consuming AsyncStorage, cli imports `react-native.config.js` and can't require `react-native-test-app` because it's a dev dependency.

Reply:
aa, got it 👍 
I'd rename `project` to maybe `exampleProjectSetup`, but that's super nitty, so ignore me 😄

Reply:
I'm going to keep it since it means we can shorthand the object below, and because you didn't feel strongly about it. :smile: